doc/sphinxman/source/glossary_psivariables.rst

```diff
@@ -97,6 +97,19 @@ PSI Variables by Alpha
    The total electronic energy [Eh] and correlation energy component [Eh]
    for the averaged coupled-pair functional level of theory.
 
+.. psivar:: ADC ROOT 0 -> ROOT m EXCITATION ENERGY - h SYMMETRY
```

Comment:
I think this should be `ADC ROOT 0 (h) -> ROOT m (i) EXCITATION ENERGY` even if adcc can't do symmetry, and h & i are always A.

But perhaps these additions were simply to keep the qcvars around when the multi-method blocks below split up?

Reply:
Exactly, the qcvars page reflects variables as they are, not variables as they'll be in a few PRs from now.